tests/Unit/App/Order/Calculator/General/CarrierSpecificCalculatorTest.php

```diff
@@ -183,6 +183,25 @@ function () {
     ],
 ]);
 
+it('calculates options for dpd', function (
+    ?callable              $orderCallback,
+    ShipmentOptionsFactory $shipmentOptions,
+    array                  $result
+) {
+    doTest(Carrier::CARRIER_DPD_NAME, $orderCallback, $shipmentOptions, $result);
+})->with([
+    'when insurance is disabled, enable it anyway' => [
```

Comment:
dit moet voor alle carriers werken, als insurance op default staat en er geen default is ingesteld oid moet er alsnog een waarde gekozen worden. zie hier al wel vaker export errors door